### ISSUE TITLE
Increase UI component size for accesibility

### ISSUE THREAD

**Issue by hectoritr** [cboard-org/cboard#974]
Based on user feedback and from some tests with a new accessibility device, we would like to increase the size and/or "clickable area" for specific components in the UI. 

We are testing an eye tracker and the precision is not enough to use the current UI size. 

**What have we tested?**
We already tried increasing the size of the UI component from:
Display -> Increase UI size
Display -> Increase Font size

We suggest that this size should be at least 66% of the size of a tile. (Only for this accessibility device, but we suggest to increase it for all users, especially the ones with motor impairments).

Buttons 1 and 2 are crucial for usability. 
<img width="1440" alt="Captura de Pantalla 2021-07-13 a la(s) 14 52 22" src="https://user-images.githubusercontent.com/12820318/125501684-313b36ed-4e66-443b-af9c-32e64b67c3ee.png">

**Comment by tomivm** [comment]
very good tests! you should try to: "enable context aware back button" to navigate between folders
Settings> Navigattion and buttons >"enable context aware back button.
and you should see these bigs buttons.
![imagen](https://user-images.githubusercontent.com/21298844/127352017-b4dc61cf-a0d0-4196-a318-113e9aff9640.png)
